src/coreComponents/mesh/generators/VTKUtilities.cpp

```diff
@@ -531,7 +534,7 @@ loadMesh( Path const & filePath,
     case VTKMeshExtension::pvti: return parallelRead( vtkSmartPointer< vtkXMLPImageDataReader >::New() );
     default:
     {
-      GEOS_ERROR( extension << " is not a recognized extension for VTKMesh. Please use .vtk, .vtu, .vtr, .vts, .vti, .pvtu, .pvtr, .pvts or .ptvi." );
+      GEOS_ERROR( extension << " is not a recognized extension for VTKMesh. Please use ." << EnumStrings< VTKMeshExtension >::concat( ", ." ) );
```

Comment:
The previous list did not contain `vtm`, the new list does. Is it intended?

Reply:
Yes, as it is managed by this `switch`, I think we should list `vtm` as a compatible extension.